mitmproxy/tools/console/consoleaddons.py

```diff
@@ -378,6 +379,8 @@ def edit_focus(self, part: str) -> None:
         # but for now it is.
         if not flow:
             raise exceptions.CommandError("No flow selected.")
+        if part in ("response-headers", "response-body", "set-cookies"):
+            flow.response = http.HTTPResponse.make()
```

Comment:
Pretty bad idea to always overwrite the response 😉

Reply:
@mhils sorry, bad idea, lost it. Give me a sec to change it.